### PR TITLE
[ASM] EXPANDR-9133 updated command filters asm-list-asset-internet-exposure

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-9133

## Description
Add filters to asm-list-asset-internet-exposure command.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py:1]: Please update `test_list_asset_internet_exposure_command` as well.

**Review by BigEasyJ — CHANGES_REQUESTED** [review]

File: Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py [hunk 1]
```diff
@@ -647,6 +657,86 @@ def list_asset_internet_exposure_command(
                 "value": [has_active_external_services],
             }
         )
+    if asm_id_list:
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py:660]: Wondering if we should create helper to clean up some of this code (and at least mandate for new additions).  Possible example:
`def append_search_param(search_params, field, operator, value):
    search_params.append({
        "field": field,
        "operator": operator,
        "value": value
    })

if asm_id_list:
    append_search_param(search_params, "asm_id_list", "in", str(asm_id_list).split(","))`

File: Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py:1]: Done!!

File: Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py (hunk 1 above)
Comment (capanw) [reply Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py:681]: Makes sense. Updated the code with this suggestion.

**Review by johnnywilkes — APPROVED** [review]
@capanw , thanks for the change and your precise attention to detail

**Comment by capanw** [comment]
@mmhw @MLainer1 approved from our side. Please review!!
Thank you

**Review by BigEasyJ — APPROVED** [review]

**Review by mmhw — APPROVED** [review]
Nice!

**Pull request merged**